Apple Windows Cleanup in AVFoundation

stwrt opened the pull request:
#### 202ede539db28000c1f6bba252d02a8e9b3205df
<pre>
Apple Windows Cleanup in AVFoundation
<a href="https://bugs.webkit.org/show_bug.cgi?id=252094">https://bugs.webkit.org/show_bug.cgi?id=252094</a>

Reviewed by Brent Fulgham.

Cleanup AVFoundation code by removing Apple Windows specific components.

* Source/WebCore/platform/graphics/avfoundation/cf/AVFoundationCFSoftLinking.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/CDMSessionAVFoundationCF.cpp:
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateAVCF.cpp: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateAVCF.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateLegacyAVCF.cpp: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateLegacyAVCF.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/MediaPlayerPrivateAVFoundationCF.cpp: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/MediaPlayerPrivateAVFoundationCF.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/WebCoreAVCFResourceLoader.cpp:

Canonical link: <a href="https://commits.webkit.org/260155@main">https://commits.webkit.org/260155@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a3ea8f7f859708347f042d0f009807dd68d00d79

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/107295 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/16352 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/40129 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/116451 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/115898 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/111194 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/17777 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/7589 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/99448 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/113060 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/13418 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/96531 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/41029 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/95319 "Passed tests") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/28166 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/82800 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/9385 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/29510 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/10027 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/6470 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/15539 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/49090 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7024 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/11580 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

brentfulgham commented on Source/WebCore/platform/graphics/avfoundation/cf/AVFoundationCFSoftLinking.h:
```diff
@@ -37,14 +37,6 @@
-#endif
-
 // Functions
 
```
I think you can delete this entire file.

stwrt replied:
Thanks updated the commit; also found a few more files that I could remove too.

brentfulgham reviewed (approved):
LGTM, but O think you can remove the entire AVFoundationSoftLink file.

stwrt committed:
Apple Windows Cleanup in AVFoundation
https://bugs.webkit.org/show_bug.cgi?id=252094

Reviewed by Brent Fulgham.

Cleanup AVFoundation code by removing Apple Windows specific components.

* Source/WebCore/platform/graphics/avfoundation/cf/AVFoundationCFSoftLinking.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/CDMSessionAVFoundationCF.cpp:
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateAVCF.cpp: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateAVCF.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateLegacyAVCF.cpp: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/InbandTextTrackPrivateLegacyAVCF.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/MediaPlayerPrivateAVFoundationCF.cpp: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/MediaPlayerPrivateAVFoundationCF.h: Removed.
* Source/WebCore/platform/graphics/avfoundation/cf/WebCoreAVCFResourceLoader.cpp:

Canonical link: https://commits.webkit.org/260155@main
